srcpkgs/minipro/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'minipro'
+pkgname=minipro
+version=0.5
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
+hostmakedepends="pkg-config libusb-devel"
+makedepends="libusb-devel which"
+depends="libusb"
+short_desc="Program for controlling the MiniPRO TL866xx series of chip programmers"
+maintainer="jleightcap <jleightcap@protonmail.com>"
+license="GPL-3.0-only"
+homepage="https://gitlab.com/natehouk/minipro"
+distfiles="https://gitlab.com/natehouk/${pkgname}/-/archive/${version}/${pkgname}-${version}.tar.gz"
+checksum=944dc3beca9bf6820bcbb7a21e9a21d8dd99f27fff4dca825a3d87f0ea062bd9
+
```

Comment:
Line 49 of their Makefile uses `which` to check for pkg-config, which needs to be added in cross build:
```
if [ "${CROSS_BUILD}" ]; then
	hostmakedepends+=" which"
fi
```
This fixes one error when I tested on aarch64-musl, but other error occurs that I'm still working out.